playwright.config.ts

```
@@ -10,11 +11,12 @@ import { defineConfig, devices } from '@playwright/test'
 const headless = !!env.CI || !!env.PLAYWRIGHT_HEADLESS
 
 const webServer: PlaywrightTestConfig['webServer'] = []
+const { ports: supabasePorts } = getSupabaseWorktreeConfig(process.cwd())
```

ESLint

[error] Unexpected use of the global variable 'process'. Use 'require("process")' instead.

(node/prefer-global/process)